mkdocs.yml

```diff
@@ -208,6 +208,7 @@ nav:
   - Debugging Analytics: "data/debugger.md"
   - Sources:
     - Overview: data/sources.md
+    - Quick Start: data/quick-start.md
```

Comment:
The quick start guide is for SDKs. I think make more sense to put under SDK folder or just use the SDKs nav bar entry for this page. Currently that entry is just a parent folder of separate SDKs no page linked to it. cc @justin-fiedler @caseyamp

Reply:
Good catch. I think it makes sense to move link the SDKs entry with the quick start. @justin-fiedler  what do you think?

Reply:
Agreed.